front/pages/w/[wId]/a/[aId]/clone.tsx

```diff
@@ -81,6 +81,7 @@ export default function CloneView({
     if (appName.length == 0) {
       setAppNameError("");
       return false;
+      // eslint-disable-next-line no-useless-escape
```

Comment:
nit: should we remove the useless escape instead ?

Reply:
Yes should be somewhat easy on this one. I presume it's the `\_` ?